extensions/sql-migration/src/constants/styles.ts

```diff
@@ -0,0 +1,70 @@
+export const secondaryText = '#605E5C';
+
```

Comment:
does the secondaryText color work for all color modes: light/dark

Reply:
thanks for bring this up. I will add the secondary text in a separate PR. This may require some additional work